src/services/cache-config/CacheConfigService.ts

```diff
@@ -387,7 +389,7 @@ export class CacheConfigService {
 
     let objectKey: string = storeType.toUpperCase().replace(/-/g, "_");
     config[storeType][objectKey]["required-node"] = true;
-    this.createHelper(builder, address.concat(storeType, objectKey), config[storeType]);
+    this.createHelper(builder, address.concat(storeType, objectKey), config[storeType], true);
```

Comment:
`createHelper` only has three params, not sure why you're passing a boolean here